services/ui-src/src/components/reports/OverlayModalPage.tsx

```diff
@@ -66,8 +78,9 @@ export const OverlayModalPage = ({
     onClose: addEditEntityModalOnCloseHandler,
   } = useDisclosure();
 
-  const openAddEditEntityModal = (entity?: EntityShape) => {
+  const openAddEditEntityModal = (entity?: EntityShape, index?: number) => {
     if (entity) setSelectedStepEntity(entity);
+    setSelectedEntityIndex(index!);
```

Comment:
And would be able to remove this line and the index param

Reply:
removed and lost in the ether